projects/plugins/boost/app/assets/src/js/features/critical-css/lib/describe-critical-css-recommendations.ts

```diff
@@ -464,6 +464,45 @@ const errorTypeSpecs: { [ type: string ]: ErrorTypeSpec } = {
 			],
 		} ),
 	},
+
+	InvalidURLError: {
+		describeSet: set =>
+			_n(
+				'Jetpack Boost found an invalid URL:',
+				'Jetpack Boost found invalid URLs:',
+				urlCount( set ),
+				'jetpack-boost'
+			),
+		suggestion: _set => ( {
+			paragraph: __(
+				'Jetpack Boost needs valid URLs from the current site in order to generate Critical CSS. It seems that one or more of the URLs you provided are invalid.',
+				'jetpack-boost'
+			),
+			list: [
+				__(
+					'Check if you have any plugins that modify your permalinks or URL structure.',
+					'jetpack-boost'
+				),
+				__(
+					'Verify your WordPress permalink settings are configured correctly in Settings → Permalinks.',
+					'jetpack-boost'
+				),
+				__(
+					'If you use custom post types, ensure they are properly registered and their permalinks are working.',
+					'jetpack-boost'
+				),
```

Comment:
Do you think we need to mention, "it is ok to ignore custom post type URLs that aren't meant to show publicly"? 🤔

Reply:
Makes sense. I've included that step and updated the second.

![CleanShot 2025-03-04 at 17 49 50](https://github.com/user-attachments/assets/59a896f9-23cf-43d7-a61f-9f797d29e360)